support-frontend/assets/components/socialShare/socialShare.jsx

```diff
@@ -59,17 +60,35 @@ export default function SocialShare(props: PropTypes) {
 
   const a11yId = `component-social-share-a11y-hint-${props.name}`;
 
+  function onShare(eventName: string, eventLink: string): () => void {
+    return (): void => {
+      let features: WindowFeatures = '';
+      if (eventName === 'contributions-share-facebook' || eventName === 'contributions-share-linkedin' || eventName === 'contributions-share-twitter') {
+        features = 'menubar=no, toolbar=no, resizable=yes, scrollbars=yes, width=500, height=400';
+      }
+      trackComponentClick(eventName);
+      window.open(
+        eventLink,
+        '',
+        features,
+      );
+    };
+  }
+
+
   return (
-    <a
+    <button
```

Comment:
I just noticed the cursor doesn't change when hovering over these buttons, which might discourage people from clicking. You'll either need to set that hover state in your CSS - or would it be possible to use storybook buttons for these too?

Reply:
I think it would be better to add social buttons to storybook instead of using the default storybook buttons for this, as we want the social icons to be bigger than the icons used next to the text